app/assets/javascripts/custom.js

```diff
@@ -13,4 +13,17 @@ $(document).ready(function(){
     slidesToShow: 4,
     slidesToScroll: 4
   });
-})
+  
+  $("#booking").prop('disabled', true);
+  $(":checkbox").on("change", function(){
+    if ($(":checkbox").is(':checked')==true)
```

Comment:
$(":checkbox").is(':checked') trả về true false rồi, không cần thiết phải so sánh

Reply:
Dùng điều kiện 3 ngôi đi